src/wifi.cpp

```diff
@@ -0,0 +1,694 @@
+// Resources:
+// - Create a basic POSIX TPC/IP server in C:
+//   https://medium.com/@coderx_15963/basic-tcp-ip-networking-in-c-using-posix-9a074d65bb35
+// - How to connect your ESP32 to Wifi and an existing POSIX TCP/IP server:
+//   https://www.youtube.com/watch?v=_dRrarmQiAM
+//   https://github.com/espressif/esp-idf/blob/v5.3.1/examples/wifi/getting_started/station/main/station_example_main.c
+//   https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/network/esp_netif.html
+//   https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/system/esp_event.html
+//   https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/network/esp_wifi.html
+
+#include "wifi.h"
+
+#if WIFI_ENABLED
+
+// Include FreeRTOS common header
+#include "freertos/FreeRTOS.h"
+// Include FreeRTOS task API
+#include "freertos/task.h"
+// Include FreeRTOS event group API
+#include "freertos/event_groups.h"
+// Import light-weight IP socket API
```

Comment:
To be consistent with the rest of the comments, change to
`// Include light-weight IP socket API`

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/edb88a636d2cf54a42f970d190cd054c04993412